test-tools/wamr-ide/VSCode-Extension/src/utilities/lldbUtilities.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2022 Amazon.com Inc. or its affiliates. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+import * as vscode from 'vscode';
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import { CheckIfDirectoryExist, DownloadFile, UnzipFile } from './directoryUtilities';
+
+const LLDB_RESOURCE_DIR = "resource/debug";
+const LLDB_OS_DIR_MAP: Partial<Record<NodeJS.Platform, string>> = {
+    "linux": "linux",
+    "darwin": "osx"
```

Comment:
is there a reason for using darwin in other places, so we don't need this map at all?